src/overlays/actors/ovl_Bg_Jya_Lift/z_bg_jya_lift.c

```diff
@@ -11,10 +11,18 @@
 #define THIS ((BgJyaLift*)thisx)
 
 void BgJyaLift_Init(Actor* thisx, GlobalContext* globalCtx);
-void BgJyaLift_Destroy(Actor* thisx, GlobalContext* globalCtx);
+//void BgJyaLift_Destroy(Actor* thisx, GlobalContext* globalCtx);
+void BgJyaLift_Destroy(BgJyaLift* this, GlobalContext* globalCtx);
```

Comment:
Destroy has to stay as taking thisx. Its an entry point function that takes a generic actor as an argument